dataBase/DBCreator.py

```diff
@@ -8,9 +8,36 @@
 # table name to schema mapping
 Schemas= {   
 'houses':
-'''
-''',
+	    '''
+	    address text primary key,
+            town text, 
+            state text,
+            status text,
+            style text,
+            category text,
+            lot text,
+            floor text,
+            rooms text,
+            bedrooms text,
+            bathrooms text,
+            basement text,
+            garage text,
```

Comment:
please add attic, rent, houseid. 
Also, there need a constraint for (address, town, state) to be unique.